src/stan-users-guide/simulation-based-calibration.Rmd

```diff
@@ -410,21 +414,21 @@ After running this for enough iterations so that the effective sample
 size is larger than $M$, then thinning to $M$ draws (here $M = 999$),
 the ranks are computed and binned, and then plotted.
 
-```{r include = TRUE, echo = FALSE, fig.align = "center", fig.cap = "Simulation based calibration plots for location and scale  of a normal model with standard normal prior on the location  standard lognormal prior on the scale.  Both histograms appear uniform, which is consistent with inference being well calibrated."}
+```{r include = TRUE, echo = FALSE, out.width="90%", fig.align = "center", fig.cap = "Simulation based calibration plots for location and scale of a normal model with standard normal prior on the location, standard lognormal prior on the scale.  Both histograms appear uniform, which is consistent with inference being well calibrated."}
```

Comment:
Later in this file you changed the figures to not use knitr, thoughts on doing it for the rest?

Reply:
eliminated.